lesson04.ipynb

```diff
@@ -0,0 +1,1158 @@
+{
```

Comment:
<p>TED talk joke is perfect, not too intrusive and is a good break to transition back from theory to reality</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>